gui/wxpython/datacatalog/tree.py

```diff
@@ -682,6 +686,35 @@ def OnCreateMapset(self, event):
                    message=_("Unable to create new mapset: %s") % e,
                    showTraceback=False)
 
+    def OnCreateLocation(self, event):
+        """
+        Location wizard started
+        """
+        grassdatabase, location, georeffile, default_region, user_mapset = \
+        create_location_interactively(self,
+                                      self.selected_grassdb[0].data['name'])
+        if location is not None:
+            self.ReloadTreeItems()
+
+            if georeffile:
+                message = _("Do you want to import <%(name)s> to the newly created location?") % {
+                    'name': georeffile}
+                dlg = wx.MessageDialog(parent=self, message=message, caption=_(
+                    "Import data?"), style=wx.YES_NO | wx.YES_DEFAULT | wx.ICON_QUESTION)
+                dlg.CenterOnParent()
+                if dlg.ShowModal() == wx.ID_YES:
+                    import_file(self, georeffile)
+                dlg.Destroy()
+
+            if default_region:
+                defineRegion = RegionDef(self, location=location)
+                defineRegion.CenterOnParent()
+                defineRegion.ShowModal()
+                defineRegion.Destroy()
+
+            if user_mapset:
+                self.OnCreateMapset(event)
```

Comment:
Seems to be exactly the case of wrong(ly used) function I mentioned:

> I can see this [`create_location_interactively()`] would be nice, but you need to move lot more functionality here to make it useful. This will also help you clean up couple calls to non-existent or wrong methods in `OnCreateLocation()` in `tree.py`.

Reply:
create_mapset_interactively used within create_location_interactively